packages/angular/projects/ui-angular/src/lib/primitives/password-field/password-field.component.ts

```diff
@@ -25,11 +26,15 @@ export class PasswordFieldComponent {
   public showPassword = false;
   public showPasswordButtonlabel = translate('Show password');
 
+  // re-export utilities so that template html can use them
+  public classnames = classnames;
+  public ComponentClassName = ComponentClassName;
+
   togglePasswordText() {
     this.showPassword = !this.showPassword;
     this.showPasswordButtonlabel = this.showPassword
-      ? translate('Show password')
-      : translate('Hide password');
+      ? translate('Hide password')
+      : translate('Show password');
```

Comment:
This was just backwords toggle for the Aria label?

Reply:
Yep, this was a bug on Angular end :(